examples/dioxus-desktop.rs

```diff
@@ -34,14 +34,8 @@ fn Body() -> Element {
 fn app() -> Element {
     use_init_i18n(|| {
         I18nConfig::new(langid!("en-US"))
-            .with_locale(Locale::new_static(
-                langid!("en-US"),
-                include_str!("./en-US.ftl"),
-            ))
-            .with_locale(Locale::new_dynamic(
-                langid!("es-ES"),
-                "./examples/es-ES.ftl",
-            ))
+            .with_locale((langid!("en-US"), include_str!("./en-US.ftl")))
+            .with_locale((langid!("es-ES"), "./examples/es-ES.ftl"))
```

Comment:
Add an import of `PathBuf` as well
```suggestion
            .with_locale((langid!("es-ES"), PathBuf::from("./examples/es-ES.ftl")))
```

Reply:
This is also wrong in the freya example btw